lib/rubocop/cop/style/redundant_filter_chain.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Identifies usages of `any?`, `empty?` or `none?` predicate methods
+      # chained to `select`/`filter` and change them to use predicate method instead.
```

Comment:
Don't forget about `find_all`. :-) Too many aliases!

Reply:
@fatkodima Not sure if you saw this comment.

Reply:
Ah, sorry, somehow missed this. Updated with this suggestion.